src/docker-in-docker/install.sh

```diff
@@ -10,6 +10,7 @@
 
 DOCKER_VERSION="${VERSION:-"latest"}" # The Docker/Moby Engine + CLI should match in version
 USE_MOBY="${MOBY:-"true"}"
+MOBY_BUILDX_VERSION="${MOBYBUILDXVERSION:-"latest"}"
```

Comment:
Can you bump the minor version for this Feature?

https://github.com/devcontainers/features/blob/f063f4ea83787b34d7c6e97c48b9c84c758a5bcd/src/docker-in-docker/devcontainer-feature.json#L3